requirements/base.in

```diff
@@ -2,46 +2,49 @@
 # like Pillow and psycopg2
 # See http://bitly.com/wheel-building-fails-CPython-35
 # Verified bug on Python 3.5.1
-wheel==0.29.0
+wheel
 
-django==1.9.4
+django
```

Comment:
Super quick tutorial:

Bumping and compiling new requirements.
* `pip-compile -U <requirements file .in>`

After adding a new requirement you don't have to bump all others, so just compiling will do:
* `pip-compile <requirements file .in>

Installing/managing requirements e.g.:
* `pip-sync requirements/production.txt`